components/css/avatar/avatar.css

```diff
@@ -0,0 +1,22 @@
+
+.avatar{
+    border-radius:50%;
+    width:48px;
+    height:48px;
+    object-fit: cover;
+}
+.avatar-sm{
+    width:32px;
+    height:32px;
```

Comment:
if possible use relative units for responsiveness

Reply:
Done. Thanks